docs/en/latest/discovery/kubernetes.md

```diff
@@ -24,12 +24,17 @@ title: Kubernetes
 ## Summary
 
 The [_Kubernetes_](https://kubernetes.io/) service discovery [_List-Watch_](https://kubernetes.io/docs/reference/using-api/api-concepts/) real-time changes of [_Endpoints_](https://kubernetes.io/docs/concepts/services-networking/service/) resources,
-then store theirs value into ngx.shared.kubernetes \
-Discovery also provides a query interface in accordance with the [_APISIX Discovery Specification_](https://github.com/apache/apisix/blob/master/docs/en/latest/discovery.md)
+then store theirs value into ngx.shared.DICT \
+Discovery also provides a node query interface in accordance with the [_APISIX Discovery Specification_](https://github.com/apache/apisix/blob/master/docs/en/latest/discovery.md)
 
-## Configuration
+## How To Use
 
-A detailed configuration for the kubernetes service discovery is as follows:
+kubernetes service discovery both support single-cluster and multi-cluster mode, \
+applicable to the case where the service is distributed in a single or multiple kubernetes clusters.
```

Comment:
```suggestion
Kubernetes service discovery both support single-cluster and multi-cluster mode, applicable to the case where the service is distributed in single or multiple Kubernetes clusters.
```

Reply:
`kubernetes` --> `Kubernetes` in this article, I remember that this is the proprietary name.